pilot/pkg/kube/inject/testdata/webhook/daemonset.yaml.injected

```diff
@@ -71,9 +68,13 @@ spec:
               fieldPath: metadata.name
         - name: ISTIO_META_INTERCEPTION_MODE
           value: REDIRECT
-        image: docker.io/istio/proxyv2:unittest
+        image: gcr.io/istio-release/proxyv2:master-latest-daily
```

Comment:
Is there a reason the introduced change has modified the HUB and TAG for all the images?

Reply:
Honestly, me and my colleague have no idea how the golden artifact files were created in the first place. My guess is because we have executed the test in the IDE and not via make. We will have a look. Thanks.